ethereum/core/src/main/java/org/hyperledger/besu/ethereum/mainnet/MainnetBlockBodyValidator.java

```diff
@@ -304,4 +308,19 @@ private boolean validateWithdrawals(final Block block) {
 
     return true;
   }
+
+  private boolean validateDeposits(final Block block) {
```

Comment:
Perhaps we need this as well, but the "Block Validity" section in the EIP is quite a different algorithm to this as it parses the transactions.

Reply:
Yes the detail of the deposit validation will be included in the next PR. For this PR I will remove any non-trivial validation on the deposit to avoid confusion